specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -1107,7 +1107,10 @@
                     "type": "object",
                     "properties": {
                       "results": {
-                        "$ref": "#/definitions/PiiResult"
+                        "$ref": "#/definitions/PiiResult",
```

Comment:
same with all the `xxResult` types https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L2242